<ui version="4.0" >
 <class>repairDialog</class>
 <widget class="QDialog" name="repairDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Dead links</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QListWidget" name="deadList" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>4</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <layout class="QHBoxLayout" >
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>37</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="selectAllDead" >
           <property name="text" >
            <string>Select all</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeDead" >
           <property name="text" >
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Activated Not Linked</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QListWidget" name="actNotLinkList" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>5</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <layout class="QHBoxLayout" >
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="selectAllActNot" >
           <property name="text" >
            <string>Select all</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="relinkActNot" >
           <property name="text" >
            <string>Re-link</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deactActNot" >
           <property name="text" >
            <string>Deactivate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Deactivated Linked</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QListWidget" name="deactLinkList" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>5</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <layout class="QHBoxLayout" >
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="selectAllDeactLink" >
           <property name="text" >
            <string>Select all</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delinkDeactLink" >
           <property name="text" >
            <string>De-link</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="activateDeactLink" >
           <property name="text" >
            <string>Activate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>271</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
